lib/api/shared/src/main/scala/net/wiringbits/api/ApiClient.scala

```diff
@@ -19,6 +20,14 @@ class ApiClient(config: ApiClient.Config)(implicit
     ex: ExecutionContext,
     sttpBackend: SttpBackend[Future, _]
 ) {
+  // Instead of using a random UUID we can avoid generating a random UUID mocking a valid UUID,
+  // this UUID is not being used in the server/api client, so it is safe to use it
```

Comment:
It's nice to leave comments but I think this is missing useful context, I'd say `While the server requires a userId, it is extracted from the Session cookie, we need a dummy value just to fulfill the method signatures`